### PR TITLE
Update set-password-to-never-expire.md

### LINKED ISSUE #12049: Article updates are incomplete
This article claims Get-MgUser is part of the AzureAD module, this is not true.

Also, if you're going to change the cmdlets used from AzureAD to Graph, as has been done in this document, you should include the profile used and the scopes required for Connect-MGGraph in order to run the listed commands. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f52c6916-7e79-9a9c-b7a6-47041ddb81e3
* Version Independent ID: 43c9b873-d354-d814-3065-1e8832361b86
* Content: [Set an individual user's password to never expire - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/add-users/set-password-to-never-expire?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/set-password-to-never-expire.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/set-password-to-never-expire.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to connect to MGgraph

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12049

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/set-password-to-never-expire.md [hunk 1]
```diff
@@ -47,8 +47,13 @@ This guide applies to other providers, such as Intune and Microsoft 365, which a
 
 ## How to check the expiration policy for a password
 
-For more information about the Get-MgUser command in the AzureAD module, see the reference article [Get-MgUser](/powershell/module/microsoft.graph.users/get-mguser).
+Use the `Connect-MgGraph` command to sign in with the required scopes. You'll need to sign in with an admin account to consent to the required scopes.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/set-password-to-never-expire.md]:
```suggestion
Use the `Connect-MgGraph` command to sign in with the required scopes. You need to sign in with an admin account to consent to the required scopes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**